server/routes/shared_api/node.py

```diff
@@ -45,3 +51,59 @@ def get_property_value(direction):
     prop = request.json['prop']
   response = fetch.raw_property_values(dcids, prop, direction == 'out')
   return Response(json.dumps(response), 200, mimetype='application/json')
+
+
+@bp.route('/propvals', methods=['GET', 'POST'])
+def expression_property_value():
+  """
+  Returns the property values for given node dcids and a relation expression for
+  a property (https://docs.datacommons.org/api/rest/v2#relation-expressions)
+  """
+  dcids = request.args.getlist('dcids')
+  if not dcids:
+    dcids = request.json['dcids']
```

Comment:
nit if this is a get request but dcids are not specified, request.json is None?

Reply:
added check